components/courses/spec/features/admin/progress/update_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe 'Progress UPDATE' do
+  describe 'validations' do
```

Comment:
very strange DSL..... you run this spec ?

Reply:
I couldn`t make tests run, so i thought, that CI will take care of it. But now, finally, i can run tests on my machine, so that i have added different tests.